v8/hybrid/README.md

```diff
@@ -0,0 +1,47 @@
+The Hybrid theme is based on the Hyde theme which is a port of the [Hyde theme by mdo](http://hyde.getpoole.com/) combined with features from the Zen theme family:
+
+* ForkAwesome font icons are supported
+* fancydates JS plugin is used
+* navigation links can be text only or icon only or combined
+* turn blog title and description in navigation bar on/off
+* Hyde color schemes are supported
+
+Specify preferences via GLOBAL_CONTEXT variable in the config:
```

Comment:
You might want to port this to v8’s `THEME_CONFIG` mechanism.

Reply:
Done.